### PR TITLE
Fix RMB tester

### PR DESCRIPTION
**Pull request by sameh-farouk**
**What's Changed:**
- Changes to the RMB Tester:
  - Fix the PythonValue error when printing the results.
  - Add a clear visual indicator for the outcome and show the percentage of failures.
  - Minor refactor.

- Changes to test_live_nodes.sh script:
  - Use `zos.system.version` instead of `rmb.version` to test live nodes (since Zos nodes no longer use the rust RMB client that used to have that command).
  - Ensure a single RMB-peer instance runs during the test.
  - Minor refactoring and improvement.

Related Issues:
- https://github.com/threefoldtech/rmb-rs/issues/206

### CODE REVIEW

**Comment by sameh-farouk** [comment]
a quick test for all networks
```
redis-server starting ..
Removes all keys in Redis
OK
rmb-peer starting (devnet)..
rmb_tester starting ..
Sending ..   |████████████████████████████████████████| 1/1 [100%] in 0.0s (34.87/s) 
Waiting ..   |████████████████████████████████████████| 31/31 [100%] in 5.1s (6.12/s) 
=======================
Summary:
=======================
sent: 1
expected_responses: 31
received_success: 31
received_errors: 0
no response errors (client give up): 0
twins not responding (twin IDs): 
elapsed time: 5.147843592996651
🎉 All responses received successfully! 🎉
=======================
stop all bash managed jobs
redis-server starting ..
Removes all keys in Redis
OK
rmb-peer starting (qanet)..
rmb_tester starting ..
Sending ..   |████████████████████████████████████████| 1/1 [100%] in 0.0s (34.84/s) 
Waiting ..   |████████████████████████████████████████| 5/5 [100%] in 1.2s (4.30/s) 
=======================
Summary:
=======================
sent: 1
expected_responses: 5
received_success: 5
received_errors: 0
no response errors (client give up): 0
twins not responding (twin IDs): 
elapsed time: 1.2429287810009555
🎉 All responses received successfully! 🎉
=======================
stop all bash managed jobs
redis-server starting ..
Removes all keys in Redis
OK
rmb-peer starting (testnet)..
rmb_tester starting ..
Sending ..   |████████████████████████████████████████| 1/1 [100%] in 0.0s (34.77/s) 
Waiting ..   |████████████████████████████████████████| 26/26 [100%] in 5.7s (4.54/s) 
=======================
Summary:
=======================
sent: 1
expected_responses: 26
received_success: 26
received_errors: 0
no response errors (client give up): 0
twins not responding (twin IDs): 
elapsed time: 5.812582165002823
🎉 All responses received successfully! 🎉
=======================
stop all bash managed jobs
redis-server starting ..
Removes all keys in Redis
OK
rmb-peer starting (mainnet)..
rmb_tester starting ..
Sending ..   |████████████████████████████████████████| 1/1 [100%] in 0.0s (32.82/s) 
Waiting ..   |███████████████████████████████████████▍⚠︎ (!) 1331/1350 [99%] in 6:40.1 (3.33/s) 
Timeout reached, stopping waiting for responses.
=======================
Summary:
=======================
sent: 1
expected_responses: 1350
received_success: 1331
received_errors: 0
no response errors (client give up): 19
twins not responding (twin IDs): 11906 4359 3596 2445 3345 5905 11156 6811 2086 12584 9012 2107 2369 8003 2891 6477 5974 7032 4860
elapsed time: 400.2224214730013
⚠ Warning: 1.41% of responses are missing! ⚠
=======================
stop all bash managed jobs
```

**Comment by ashraffouda** [comment]
tried it from ur branch on  devenet and testnet and it gives 0 successful responses,can u recheck it again maybe something wrong from my side
devnet
image
testnet
image

**Comment by sameh-farouk** [comment]
In your screenshots, the rmb-peer binary did not start, which is the reason for the issue. What is the content of the log file?

**Comment by ashraffouda** [comment]
ok, it was my fault I exported MNEMONICS instead of MNEMONIC I suggest u fail with a clear msg if MNEMONIC="" 
something like what u r doing if the script started without passing a network 
image

**Review by ashraffouda — APPROVED** [review]

File: tools/rmb_tester/test_live_nodes.sh [hunk 1]
```diff
@@ -36,21 +45,32 @@ trap cleanup SIGHUP	SIGINT SIGQUIT SIGABRT SIGTERM
...
 set -e
 
+# ensure that RMB is not already running
+if pgrep -x $(basename "$RMB_BIN") > /dev/null; then
+    echo "Another instance of rmb-peer is already running. Killing..."
+    pkill -x $(basename "$RMB_BIN")
+fi
+
+# ensure the MNEMONIC has no leading or trailing spaces
+MNEMONIC="${MNEMONIC#"${MNEMONIC%%[![:space:]]*}"}"; MNEMONIC="${MNEMONIC%"${MNEMONIC##*[![:space:]]}"}"
```
Comment (ashraffouda) [inline tools/rmb_tester/test_live_nodes.sh:62]: I suggest u fail here if MNEMONIC = "", because the result is not clear in this case

**Review by ashraffouda — CHANGES_REQUESTED** [review]

File: tools/rmb_tester/test_live_nodes.sh (hunk 1 above)
Comment (sameh-farouk) [reply tools/rmb_tester/test_live_nodes.sh:72]: Done.
Additionally, there was an issue where the script continued running after the cleanup function executed. Essentially, the cleanup function was called after the rmb-peer start timeout within a subshell, so the exit command in the cleanup function did not terminate the main script, and it continued to run.

The script is now capable of properly handling both errors.

**Review by ashraffouda — APPROVED** [review]

**Comment by ashraffouda** [comment]
nice

**Pull request merged**